flink-table/flink-table-planner/src/main/java/org/apache/flink/table/calcite/CalciteParser.java

```diff
@@ -47,7 +47,7 @@ public SqlNode parse(String sql) {
 			SqlParser parser = SqlParser.create(sql, config);
 			return parser.parseStmt();
 		} catch (SqlParseException e) {
-			throw new SqlParserException("SQL parse failed. " + e.getMessage());
+			throw new SqlParserException("SQL parse failed." + e.getMessage(), e);
```

Comment:
seems removing a space by mistake

Reply:
Good catch @libenchao . I fixed it while merging.

Reply:
Good catch @libenchao. Thank you. I fixed it while merging.